tests/phpunit/elementor/core/breakpoints/test-breakpoint.php

```diff
@@ -0,0 +1,149 @@
+<?php
+namespace Elementor\Tests\Phpunit\Elementor\Core\Breakpoints;
+
+use Elementor\Core\Breakpoints\Breakpoint;
+use Elementor\Core\Breakpoints\Manager as Breakpoints_Manager;
+use Elementor\Plugin;
+use Elementor\Testing\Elementor_Test_Base;
+use Elementor\Testing\Traits\Extra_Assertions;
+
+class Test_Breakpoint extends Elementor_Test_Base {
+
+	use Extra_Assertions;
+
+	/**
+	 * Test Get Name
+	 *
+	 * Tests Breakpoint::get_name() for all breakpoints.
+	 *
+	 * @since 3.2.0
+	 */
+	public function test_get_name() {
+		$breakpoints = Plugin::$instance->breakpoints->get_breakpoints();
+
+		foreach ( $breakpoints as $breakpoint_name => $breakpoint ) {
+			/** @var Breakpoint $breakpoint */
+			$this->assertEquals( $breakpoint_name, $breakpoint->get_name() );
+		}
+	}
+
+	/**
+	 * Test Get Config
+	 *
+	 * Tests Breakpoint::get_config() for all breakpoints.
+	 *
+	 * @since 3.2.0
+	 */
+	public function test_get_config() {
+		$breakpoints = Plugin::$instance->breakpoints->get_breakpoints();
+
+		foreach ( $breakpoints as $breakpoint ) {
+			/** @var Breakpoint $breakpoint */
+			$breakpoint_config = $breakpoint->get_config();
+
+			$this->assertArrayHaveKeys( [ 'value', 'direction', 'is_enabled' ], $breakpoint_config );
+			$this->assertTrue( is_numeric( $breakpoint_config['value'] ) );
```

Comment:
```suggestion
			$this->assertTrue( is_int( $breakpoint_config['value'] ) );
```

Reply:
the value can be fetched from the DB as a string

Reply:
Cast the value to int when initializing value from DB

Reply:
Done.